README.md

```diff
@@ -97,6 +97,13 @@ To build with lua instead of luaJIT, run:
 $ ./x.py build -DENABLE_LUAJIT=OFF
 ```
 
+To build with more detail debug symbols, run:
+
+```shell
+# Default build type is `RelWithDebInfo` and optimization level is `-O2`. If you want more detail debug info, you can change it to `Debug` by:
+$ ./x.py build -DCMAKE_BUILD_TYPE=Debug
+```
+
```

Comment:
```suggestion
Build with the development mode, run:

```shell
# The default build type is `RelWithDebInfo` and its optimization level is `O2`.
# You can change it to `Debug` in development mode.
$ ./x.py build -DCMAKE_BUILD_TYPE=Debug
```

```